src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.css">
+    <!-- <link rel="stylesheet" href="./styles/stars.css"> -->
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__image" 
+        src="./images/imac.jpeg" 
+        alt="imac"
+      >
+      <div class="card__information">
+        <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__ratings">
+          <div class="card__stars-list">
+            <ul class="stars--4 stars">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+          </div>
+          <div class="card__review">
+            <p>Reviews: 5</p> 
+          </div>
+        </div>
+        <div class="card__price price">
+          <div class="price__label">
+            <p>Price:</p>
+          </div>
```

Comment:
no need to wrap <p> into separate <div>
you may use `display: block` if you need it